assets/app/jsx/AdminPanel/createPagination.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const Pagination = ({ postsPerPage, totalPosts, paginate }) => {
+  const pageNumbers = [];
+
```

Comment:
what if total pages is a big number (eg: gt 10) then its not good to show  more than 10 tabs for pagination... 
we should show something like 10tabs and two tabs (left and right) for previous and next tabs.
[example](https://miro.medium.com/max/638/1*F4X8aQyw3Ik7iINPg4AAbQ.png
)